### PR TITLE
More sections

### PR DESCRIPTION
**Pull request by kevinzwang**
https://deep-blue-training.readthedocs.io/en/more-sections/

### CODE REVIEW

**Comment by kevinzwang** [comment]
Don't worry about the CI, the `build-test` check is supposed to fail

File: docs/section-5/writing-tests.md [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+    2. Line 25 of `MysteryMath.java`, in `inverseAbs()`, should absolute value the input if it's less than zero, not greater.
+
+## MysteryMath2
+
+Now, it's your time to write some tests. The `quadraticFormula()` method in `MysteryMath2.java` looks like it works, or does it? Write `MysteryMath2Test.java` to test the code with different coefficients. What inputs make the tests fail? Once you figure it out, you should also fix the code.
+
+??? success "Click for answers"
+    1. If your test inputs coefficients for an equation that has no solutions (ex: `x^2+x+1`), the program is going to throw an exception.
+    2. To fix this, the `quadraticFormula()` method should first check if `b*b-4*a*c` is non-negative before doing the calculation. If it's negative, it should just return -1.
+
```
Comment (brettle) [inline docs/section-5/writing-tests.md:79]: All of this is great, but I'd like to see something like a "When to write tests" section that encourages students to write tests before they write the code under test. An exercise similar to the quadraticFormula() one above would work well.

File: mkdocs.yml [hunk 2]
```diff
@@ -42,9 +40,19 @@ nav:
...
-  - 5. Advanced Programming:
+  - 5. Unit Testing:
     - Introduction: section-5/intro.md
-    - Additional Topics: section-5/additional-topics.md
+    - Why Unit Tests?: section-5/why-tests.md
+    - Writing Tests: section-5/writing-tests.md
+    - Continuous Integration: section-5/ci.md
+  - 6. Build Season Process:
+    - Introduction: section-6/intro.md
+    - Project Organization: section-6/organization.md
```
Comment (brettle) [inline mkdocs.yml]: I suggest renaming this to something like "Branch Organization". The term "Project Organization" is usually used to refer to organization of the code into files and folders.

File: docs/section-6/delegation.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
+## Trello
+
+We will use Trello for task delegation. Please read through [this presentation](https://docs.google.com/presentation/d/1E6NWQwGugOpqZyM3Rbugq30OSHSfIXFvhhhAqgV1Mbk/edit#slide=id.g255a686be2_0_25) starting from the page that says "Project Management with Trello", and then fill out the form on the last page to get onto the team Trello.
+
+More about how the programming team uses Trello:
+
+- The programming lead sorts the cards in the "Todo" list by priority, but anyone is welcome to add cards to the bottom of the list.
+- The programming lead will assign tasks to team members. If you have not been assigned a task, you can choose the highest card on the "Todo" list that you can do.
+- Assign yourself to a card when you start it, and move it to the "In Progress" column. When you are done, move it to the "Done" columm and notify a programming lead or mentor to check you off and archive the card.
```
Comment (brettle) [inline docs/section-6/delegation.md]: columm -> column

File: docs/section-6/pull-requests.md [hunk 4]
```diff
@@ -0,0 +1,10 @@
+## Code Review Process
+Feature branches should go through two code reviews:
+
+1. The first one where there is no no new implementation. All code here is either structural code (classes and methods setup but not implemented), or code that we have already written in the past (such as drivetrain code). This is to catch structural problems early on and have the reviewer familiarize themself with the code so it will be easier to do the final review.
```
Comment (brettle) [inline docs/section-6/pull-requests.md]: no no -> no

**Review by brettle — APPROVED** [review]

File: docs/section-5/writing-tests.md (hunk 1 above)
Comment (kevinzwang) [reply docs/section-5/writing-tests.md:79]: I don't think an exercise is necessary but I will add a part that says to write tests before code

**Pull request merged**